ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -669,6 +671,49 @@ export default class AdvancedTab extends PureComponent {
     );
   }
 
+  renderTokenDetectionToggle() {
+    if (!process.env.TOKEN_DETECTION_V2) {
+      return null;
+    }
+
+    const { t } = this.context;
+    const { useTokenDetection, setUseTokenDetection } = this.props;
+
+    return (
+      <div
+        ref={this.settingsRefs[4]}
```

Comment:
NIT: Looks like there is no existing organization to these render functions. I think it would help readability to reorganize these render functions by their display order or alphabetically. This would be out of the scope of this PR. Could be nice to move this near 3 and 5 in the meantime

Reply:
I have reverted the index changes and changed the index for the new function to the latest.